a11ym

```diff
@@ -170,7 +172,7 @@ var tester = Tester(program);
 // When the maximum number of URL is reached, then we kill this queue, in
 // addition to kill all URL queues.
 var testQueue = async.queue(
-    function (task, onTaskComplete) {
+    function (url, onTaskComplete) {
         if (--maximumUrls < 0) {
             killQueue('Maximum URLs reached.');
             onTaskComplete();
```

Comment:
You must call `onTaskComplete` before `killQueue`, else the number returned by `testQueue.running` is wrong (from the user perspective).